WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerActivity.kt

```diff
@@ -162,9 +153,27 @@ class StoryComposerActivity : ComposeLoopFrameActivity(),
         }
 
         editorMedia.start(requireNotNull(site), this, STORY_EDITOR)
+        setupEditorMediaObserver()
+        setupViewModelObservers()
+    }
+
+    private fun setupViewModelObservers() {
+        viewModel.mediaFilesUris.observe(this, Observer { uriList ->
+            addFramesToStoryFromMediaUriList(uriList)
```

Comment:
This might be planned for future refactoring, so feel free to ignore this comment.... We might want to consider moving both these methods into the VM. Wdyt?

Reply:
Thanks for this comment. Yes, it would be good to move these methods into a VM. They are located within the `ComposeLoopFrameActivity` from the portkey project so when refactoring is being done there I assume that would happen then.